client/src/services/plugins/ExcelImportPlugin.js

```diff
@@ -55,6 +59,8 @@ class StandardExcelImportFormat {
 
         const parent = row[headers[RelationshipFromColumn]];
         const relationship = row[headers[RelationshipTypeColumn]];
+        const initRelationshipDataRaw = row[headers[RelationshipInitialDataColumn]];
```

Comment:
What happens to existing files? I.e. that don't use the new format. Won't this line break?

I don't think we want to break import of existing files. We should probably check for the existence of this new column before pulling out data.

Reply:
Trying to read the new col from an old file without the new col will return an "empty" object and send it over "empty" to the addRelations in the API service, the same as any row not having a value for relationship properties.

This did required a Fix but on line 103 of the same file since we were validating the file cols match the ColumnMapping Object even though the only required column is the IdColumn and we already have a validation in row 40. Existing and New Files are now supported with this fix.